components/build/tekton-results/allow-argocd-to-manage.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: grant-argocd
+  namespace: tekton-pipelines
```

Comment:
If Tekton is installed via a `Subscription` resource, it's deployed under `openshift-pipelines`. The namespace `tekton-pipelines` is only relevant if you're installing tekton directly from upstream. Is my understanding correct?

Reply:
Yes, we are installing `tekton-results` which are not part of `openshift-pipelines`. `tekton-results` are installed into `tekton-pipelines` and also having pointers to `tekton-pipelines` so that's why there are kept in `tekton-pipelines` namespace.